templates/front/list-portfolio.tpl

```diff
@@ -8,7 +8,7 @@
 
         <div class="ia-item__content">
             <h4 class="ia-item__title text-center">
-                <a href="{$entry.link}">{$entry.title|escape}</a>
+                <a href="{$entry.item}/{$entry.category_alias}/{$entry.id}-{$entry.title_alias}.html">{$entry.title|escape}</a>
```

Comment:
Revert back these changes. Every item has processItem method now. You should generate View link there. Not a way to go.

Contact @Batry to discuss the details.